### PR TITLE
[Quantum] Update version and history info for az quantum extension 0.18.0

### PR DESCRIPTION
**Pull request by warren-jones**
Release PR for az quantum extension, version 0.18.0.

---

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

### CODE REVIEW

File: src/quantum/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+0.18.0
+++++++
+* [2023-02-07] Version intended to work with QDK version 0.27.253010 and Azure CLI 2.41.0 or greater.
+* You can now submit QIR, QIO, and pass-through jobs using the CLI.
```
Comment (shenyingjun) [inline src/quantum/HISTORY.rst:9]: QIO is not fully supported. let's not mention that in the note

**Review by shenyingjun — APPROVED** [review]

File: src/quantum/HISTORY.rst (hunk 1 above)
Comment (warren-jones) [reply src/quantum/HISTORY.rst]: Revised the QIO support note

Comment (warren-jones) [reply src/quantum/HISTORY.rst]: Deleted mention of QIO

**Comment by yonzhan** [comment]
Quantum

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**